packages/xchain-monero/src/walletRpc.ts

```
@@ -0,0 +1,334 @@
+  }>(url, 'get_transfers', {
+    in: true,
+    out: true,
+    pending: false,
+    failed: false,
+    pool: false,
+    account_index: accountIndex,
+  })
+
+  return [...(result.in ?? []), ...(result.out ?? [])]
+    .map(normalizeTransfer)
+    .filter((tx) => tx.txid && tx.height > 0)
```

ESLint

[error] Replace `⏎····.map(normalizeTransfer)⏎····` with `.map(normalizeTransfer)`

(prettier/prettier)